### PR TITLE
fix integration tests on jenkins

### PR DESCRIPTION
**Pull request by francois-vincent**
TODO:

- [ ] check what's wrong with RabbitMQ when testing Kraken using Jenkins

### CODE REVIEW

File: integration_tests/test_common/test_kraken.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def _test_test_kraken_nowait_nofail(platform, capsys, map, ret_val):
         assert fabric.execute('test_kraken', krak, fail_if_error=False).values()[0] is ret_val
         out, err = capsys.readouterr()
         assert 'http://{}:80/monitor-kraken/?instance={}'.format(platform.get_hosts()[host], krak) in out
+        # TODO fix on Jenkins
```
Comment (pbougue) [inline integration_tests/test_common/test_kraken.py]: Why not pushing the real code?
The goal is to save it and to set a target, in my mind.

Comment (francois-vincent) [inline integration_tests/test_common/test_kraken.py]: there's no real code yet. I must think about it.
And I need the other PR to be merged :)

Comment (pbougue) [inline integration_tests/test_common/test_kraken.py]: I was mentioning the code you wrote on the other PR.

**Comment by Hoshiyo** [comment]
Jenkins, test this please

**Comment by Hoshiyo** [comment]
Jenkins, test this please

**Pull request closed without merging**